man/systemd.network.xml

```diff
@@ -796,6 +796,52 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[IPRule] Section Options</title>
+
+      <para>An <literal>[IPRule]</literal> section accepts the
+      following keys. Specify several <literal>[IPRule]</literal>
+      sections to configure rules in the routing policy database control the
+       route selection algorithm. </para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>TypeOfService=</varname></term>
+          <listitem>
+            <para> Specifies type of service.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>From=</varname></term>
+          <listitem>
+            <para>Specifies the source prefix to match is an address with a prefix length, separated by a slash
+            <literal>/</literal> character. </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>To=</varname></term>
+          <listitem>
+            <para>Specifies the destination prefix to match is an address with a prefix length, separated by a slash
+            <literal>/</literal> character.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>FWMark=</varname></term>
+          <listitem>
+            <para>Specifies the fwmark value to match.</para>
```

Comment:
We need some explanation how this mark is set. Maybe a reference to ip man page or something.